changelogs/fragments/update_requires_ansible.yaml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

This can probably be `trivial` instead
```suggestion
trivial:
```

---

Fixed.